mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -276,6 +279,7 @@ impl TendermintActivationPolicy {
 
                 #[cfg(target_arch = "wasm32")]
                 PrivKeyPolicy::Metamask(_) => unreachable!(),
+                PrivKeyPolicy::WalletConnect { .. } => unreachable!(),
```

Comment:
Q: Is it really impossible to get the public key from WalletConnect?

Reply:
you mean privateKey?, No

Reply:
I read it wrong